### ISSUE TITLE
✨ New job poster template page

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#11434]
## ✨ Feature

A page that shows all the details of a job poster template

## 🕵️ Details

New page title: Job template: `template_name`
URL: `/job-templates/#id`  

Out of scope for this issue: The download links in the left menu.
![image](https://github.com/user-attachments/assets/f572588b-a490-4668-ba70-60f21ec099c7)

### Basic details
- Just displaying the data that's in the model.
- The "view on GCpedia" link should be the "Work description URL" from the data model (English or French) - note that these links will be "internal" to gov, so might not be able to easily test the ones provided, but I think that's ok. We can test on UAT (in Gray's oppinion)
- The note at the bottom should always be there
![image](https://github.com/user-attachments/assets/cd5b7588-fe29-429a-8dc4-0c6d783db146)

### Key task examples
- The circled text in the image below is static
- The key tasks are pulled from the data model, but we need to make sure they display like in this design. In the future these will be populated from a Rich text input (that allows for formatting). More details on this in #11435.  
![image](https://github.com/user-attachments/assets/c9d5c901-35ac-4f16-a248-8954c14a6465)

### Skills sections

There are 3 skills sections that need to be dynamically populated based on the skills associated with the job template.

**UPDATE:** These skill sections should use the same component to list skills as the job advertisement, though without showing assessment time, Required/Asset, and possibly level.

**Essential Technical skills - Examples**
- Copy at the top is static
- All **technical** skills from the list of skills associated with job template that are tagged as "Essential examples" should be displayed in this first list along with their corresponding "skill levels". The mapping (number to level) for these skill levels is described further in #11435.
- Sort the skills by skill level - then alphabetically (even if they are not sorted like that in the design)
- The optional note section at the bottom is only shown if "Essential technical skill notes" is not empty (or not null, however is best)

**Essential behavioural skills - Examples**
- Copy at the top is static
- All **behavioural** skills from the list of skills associated with job template that are tagged as "Essential examples" should be displayed in this second list along with their corresponding "skill levels". The mapping (number to level) for these skill levels is described further in #11435.
- Sort the skills by skill level - then alphabetically (even if they are not sorted like that in the design)
- Similar to the design for "Essential technical skill notes", there can be notes for this section as well if "Essential behavioural skill notes" is not empty (or not null, however is best)

**Asset technical skills - Examples**
- Copy at the top is static
- All **technical** skills from the list of skills associated with job template that are tagged as "Asset examples" should be displayed in this third list. There are NO skill levels displayed for these asset skills.
- Sort the skills alphabetically (even if they are not sorted like that in the design)
- Similar to the design for "Essential technical skill notes", there can be notes for this section as well if "Asset technical skill notes" is not empty (or not null, however is best)

## 🎨 Design

@JoshBeveridge designed this one

https://www.figma.com/design/kTvZVVsdVziRqxanyUgSKZ/Job-template-library?node-id=355-21116&node-type=FRAME&t=pnKV4IMENftZID59-0

## 📸 Screenshot

![Image](https://github.com/user-attachments/assets/e5e87bc3-10e7-4db5-baf7-08a9dd2bce27)

## ✅ Acceptance Criteria

- [ ] Create page to match design
- [ ] **On this page** menu
- [ ] Basic details section
- [ ] Key tasks section
- [ ] 3 skills sections: Essential technical, Essential behavioural, Asset technical
  - skill lists use accordion components, ideally same component as on Job Poster, though without showing assessment time, Required/Asset, and without showing skill level for asset technical skills.
  - skill accordion lists need to handle an optional footer, which shows the "skill notes" field if non-empty
- [ ] Out of scope for this issue: The download links in the left menu.

## 🛑 Blockers

```[tasklist]
### Blocked By
- [ ] https://github.com/GCTC-NTGC/gc-digital-talent/issues/11429
```
